spec/fixtures/gradle-kotlin-dependencies-q.txt

```diff
@@ -0,0 +1,312 @@
+
```

Comment:
FTR, this file was generated via `gradle init`, and selecting a Kotlin application with the Kotlin gradle DSL.